### ISSUE TITLE
Time Picker hour/minute picker overlay is getting cut off when embedded in a tab content 

### ISSUE THREAD

**Issue by monterey01** [jdewit/bootstrap-timepicker#146]
Time Picker hour/minute picker overlay is getting cut off when embedded in a tab content 

You can recreate the issue at http://env-1124382.jelastic.servint.net/restaurant/?timezone=Etc/GMT0. Click on the  Add Open Close Times button on the monday tab and the issue can be recreated

<div id="my-tab-content" class="tab-content">
        <div class="tab-pane active" id="red">
            <div class="control-group">
        
                <div class="controls form-inline">
                    <label>Open Time</label>
                    <div class="input-append bootstrap-timepicker">
                        <input id="timepicker1" type="text" class="input-small"> <span
                            class="add-on"><i class="icon-time"></i></span>
                        <script type="text/javascript">
                            $('#timepicker1').timepicker();
                        </script>

                    </div>
                    <label>Close Time</label>
                    <div class="input-append bootstrap-timepicker">
                        <input id="timepicker2" type="text" class="input-small"> <span
                            class="add-on"><i class="icon-time"></i></span>
                        <script type="text/javascript">
                            $('#timepicker2').timepicker();
                        </script>

                    </div>
                </div>
            </div>
        </div>

Complete JSP Page->

<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
<%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt"%>
<%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
<jsp:useBean id="timeZoneList" type="java.util.List" scope="request" />
<jsp:useBean id="currencyCodeList" type="java.util.List" scope="request" />

<!DOCTYPE html>

<html class="no-js">

<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<title></title>
<meta name="description" content="">
<meta name="viewport" content="width=device-width">

<link rel="stylesheet"
    href="<c:url value='/resources/css/bootstrap.min.css'/>">
<style>
body {
    padding-top: 60px;
    padding-bottom: 40px;
}
</style>

<script
    src="//ajax.googleapis.com/ajax/libs/jquery/1.10.1/jquery.min.js"></script>
<link rel="stylesheet"
    href="<c:url value='/resources/css/bootstrap-responsive.min.css'/>">
<link rel="stylesheet" href="<c:url value='/resources/css/main.css'/>">
<link rel="stylesheet"
    href="<c:url value='/resources/css/bootstrap-timepicker.min.css'/>">
<script src="<c:url value='/resources/js/vendor/bootstrap.min.js'/>"></script>

<script
    src="<c:url value='/resources/js/vendor/modernizr-2.6.2-respond-1.1.0.min.js'/>"></script>

<script src="<c:url value='/resources/js/main.js'/>"></script>

<script
    src="<c:url value='/resources/js/vendor/bootstrap-timepicker.min.js'/>"></script>

<script>
    var _gaq = [ [ '_setAccount', 'UA-XXXXX-X' ], [ '_trackPageview' ] ];
    (function(d, t) {
        var g = d.createElement(t), s = d.getElementsByTagName(t)[0];
        g.src = '//www.google-analytics.com/ga.js';
        s.parentNode.insertBefore(g, s);
    }(document, 'script'));
</script>

</head>
<body>

    <script>
        $(function() {
            $('#name').focus();
        });
        function enableDisableOnlineOrderingOptions(onlineOrderingAllowed) {

            if ($(onlineOrderingAllowed).is(':checked')) {
                $('#maxOrderAmount-div').show();
                $('#maxOrderAmount').focus();
            }

            else {
                $('#maxOrderAmount-div').hide();
            }
        }
    </script>

    <form:form id="generateURLInfo" action="/restaurant/submitRestaurant"
        commandName="restaurant" method="post" class="form-horizontal">

        <fieldset>

            <form:errors path="*" cssClass="alert alert-error" element="div"
                delimiter='</div><div class="alert alert-error">' />

            <spring:bind path="name">
                
                <div class="control-group ${status.error ? 'error' : '' }">
                    <form:label class="control-label  " path="name">Name </form:label>
                    <div class="controls">
                        <form:input id="name" path="name" placeholder="Restaurant Name"
                            class="input-xlarge" />
                        <c:if test="${status.error}">
                            <span class="help-inline">${status.errorMessage}</span>
                        </c:if>
                    </div>
                </div>
            </spring:bind>

            
            <div class="control-group">
                <label class="control-label">Time Zone</label>
                <div class="controls">
                    <form:select path="timeZone" class="input-xlarge">
                        <c:forEach var="tmp" items="${timeZoneList}">
                            <form:option value="${tmp}">
                                <spring:message code="${tmp}" />
                            </form:option>
                        </c:forEach>
                    </form:select>

                </div>
            </div>

            
            <div class="control-group">
                <label class="control-label">Currency</label>
                <div class="controls">
                    <form:select path="currency" class="input-xlarge">
                        <c:forEach var="tmp" items="${currencyCodeList}">
                            <form:option value="${tmp}">
                                <spring:message code="currency.key.${tmp}" />
                            </form:option>
                        </c:forEach>
                    </form:select>

                </div>
            </div>

            
            <div class="control-group">
                <label class="control-label">Online Ordering Allowed</label>
                <div class="controls">
                    <label class="checkbox"> <form:checkbox
                            path="onlineOrderingAllowed"
                            onClick="enableDisableOnlineOrderingOptions(this)" />
                    </label>
                </div>
            </div>

            
            <div id="maxOrderAmount-div" class="control-group"
                style="display: none">
                <form:label class="control-label" path="maxOrderAmount">Maximum Order Amount</form:label>
                <div class="controls">
                    <form:input id="maxOrderAmount" path="maxOrderAmount"
                        placeholder="placeholder" class="input-xlarge" />

                </div>
            </div>

            <script>
                function makeDayActiveInactive(inputElement) {

                    if ($(inputElement).hasClass('active')) {

                        $(inputElement).removeClass('active');

                    }

                    else {

                        $(inputElement).addClass('active');
                    }

                }
            </script>

            <div class="control-group">
                <label class="control-label">Hours Open</label>
                <div class="controls">
                    <c:forEach items="${days}" var="day">
                        <button class="btn btn-s"
                            onClick="event.preventDefault()">${day}</button>

                        <script type="text/javascript">
                            $('#timepicker1').timepicker();
                        </script>
                    </c:forEach>
                </div>
            </div>

            
            <div id="content">
    <ul id="tabs" class="nav nav-pills" data-tabs="tabs">
        <li class="active"><a href="#red" data-toggle="tab">Red</a></li>
        <li><a href="#orange" data-toggle="tab">Orange</a></li>
        <li><a href="#yellow" data-toggle="tab">Yellow</a></li>
        <li><a href="#green" data-toggle="tab">Green</a></li>
        <li><a href="#blue" data-toggle="tab">Blue</a></li>
    </ul>
    <div id="my-tab-content" class="tab-content">
        <div class="tab-pane active" id="red">
            <div class="control-group">
        
                <div class="controls form-inline">
                    <label>Open Time</label>
                    <div class="input-append bootstrap-timepicker">
                        <input id="timepicker1" type="text" class="input-small"> <span
                            class="add-on"><i class="icon-time"></i></span>
                        <script type="text/javascript">
                            $('#timepicker1').timepicker();
                        </script>

                    </div>
                    <label>Close Time</label>
                    <div class="input-append bootstrap-timepicker">
                        <input id="timepicker2" type="text" class="input-small"> <span
                            class="add-on"><i class="icon-time"></i></span>
                        <script type="text/javascript">
                            $('#timepicker2').timepicker();
                        </script>

                    </div>
                </div>
            </div>
        </div>
        <div class="tab-pane" id="orange">
            <h1>Orange</h1>
            <p>orange orange orange orange orange</p>
        </div>
        <div class="tab-pane" id="yellow">
            <h1>Yellow</h1>
            <p>yellow yellow yellow yellow yellow</p>
        </div>
        <div class="tab-pane" id="green">
            <h1>Green</h1>
            <p>green green green green green</p>
        </div>
        <div class="tab-pane" id="blue">
            <h1>Blue</h1>
            <p>blue blue blue blue blue</p>
        </div>
    </div>
</div>

```
        <!-- Button -->
        <div class="control-group">
            <div class="controls">
                <form:button class="btn btn-primary" type="submit"> Submit</form:button>
            </div>
        </div>
    </fieldset>
</form:form>
```

</body>
</html>

![image](https://f.cloud.github.com/assets/3004643/1031207/62169a3c-0ed6-11e3-8046-ca9b7530fa16.png)

**Comment by jdewit** [comment]
just increase the z-index on the dropdown-menu or set a min height on your tab-content.
